HunterPie.Integrations/Datasources/MonsterHunterRise/Entity/Player/MHRPlayer.cs

```diff
@@ -654,6 +721,9 @@ private void GetPlayerWirebugs()
                     AddressMap.GetAbsolute("UI_ADDRESS"),
                     AddressMap.Get<int[]>("IS_WIREBUG_BLOCKED_OFFSETS")
                 ) != 0,
+                WirebugCondition = _commonCondition.HasFlag(CommonConditions.WindMantle) ? CommonConditions.WindMantle.ToString()
+                                 : _debuffCondition.HasFlag(DebuffConditions.IceL) ? DebuffConditions.IceL.ToString()
+                                 : "None",
```

Comment:
This isn't good, it should be an enum and not a string, strings are prone to error and having to hardcode strings everywhere that I need to access this WirebugCondition is bad.
Another reason why having strings as state is bad is because strings are allocated on the heap, creating Garbage Collection overhead, this code is pretty much allocating a new string on the heap every single scan.